spec/support/vcr_cassettes/okta/connection_logs.yml

```diff
@@ -0,0 +1,143 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://deptva-eval.okta.com/api/v1/logs?filter=eventType%20eq%20%22app.oauth2.as.consent.grant%22&since=2020-12-23T19:47:05Z
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - Faraday v0.17.0
+      Content-Type:
+      - application/json
+      Accept:
+      - application/json
+      Authorization:
+      - SSWS <OKTA_TOKEN>
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Date:
+      - Mon, 28 Dec 2020 19:47:06 GMT
+      Content-Type:
+      - application/json
+      Transfer-Encoding:
+      - chunked
+      Connection:
+      - keep-alive
+      Server:
+      - nginx
+      Public-Key-Pins-Report-Only:
+      - pin-sha256="r5EfzZxQVvQpKo3AgYRaT7X2bDO/kj3ACwmxfdT2zt8="; pin-sha256="MaqlcUgk2mvY/RFSGeSwBRkI+rZ6/dxe/DuQfBT/vnQ=";
```

Comment:
There's no secrets in these here cassettes? Right? RIGHT?

Reply:
it's just the Public Key Information, and is in every other cassette on vets-api. So as far as I know it's fine.
![Screen Shot 2021-02-23 at 3 55 52 PM](https://user-images.githubusercontent.com/27251568/108913474-9c02e100-75ef-11eb-8e54-143c27bb4033.png)